osf/migrations/0224_population_registration_subscription_notifications.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-10-26 18:43
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from osf.migrations import populate_registration_provider_subscriptions
+
+
+def revert(apps, schema_editor):
+    NotificationSubscription = apps.get_model('osf', 'NotificationSubscription')
+    # The revert of this migration deletes all NotificationSubscription instances
+    NotificationSubscription.objects.filter(provider__isnull=False).delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0223_auto_20201026_1843'),
+    ]
+
+    operations = [
+        migrations.RunPython(populate_registration_provider_subscriptions, revert)
```

Comment:
Is this necessary given that there's a `populate_registration_provider_notification_subscriptions.py` management command that we could run after deployment? Or rather, does this make the management command unnecessary?